Remove liteCheckpointFATs step as this will now be covered by the normal FAT step

rickettmwork opened the pull request:
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

################################################################################################

rickettmwork committed:
Remove liteCheckpointFATs step as this will now be covered by the normal FAT step

rickettmwork commented:
!pb@dev
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=a90829f2-cf0d-4d84-adde-02a7180170a2) - :x: completed with errors/failures.)
limit.fat.buckets.to.feature: checkpoint

_Note: Target locations of links might be accessible only to IBM employees._

qas93 reviewed (approved):
Approved given successful pb

websterc87 reviewed (changes requested):
By removing the explicit checkpoint step we will stop a number of the checkpoint fats running as checkpoint has never been subject to the same AUTO mode discovered rules,
We either need to get agreement that we can run dramatically fewer checkpoint fats or we need to introduce new logic to support running them always as is currently the case.

rickettmwork commented:
Needs further discussion, however PB showed no issues

rickettmwork commented:
!pb@dev ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=d5824fb9-cb65-4550-b3ed-3b6276531196) - :x: completed with errors/failures.)

_Note: Target locations of links might be accessible only to IBM employees._

rickettmwork commented:
> By removing the explicit checkpoint step we will stop a number of the checkpoint fats running as checkpoint has never been subject to the same AUTO mode discovered rules, We either need to get agreement that we can run dramatically fewer checkpoint fats or we need to introduce new logic to support running them always as is currently the case.


We have addressed these concerns as follows:
- the checkpoint fats are now running properly in continuous and release pipelines so we don't lose that quality gate
- we've updated the ebc ansible to install criu from binaries so the performance concern is fixed
- a quick search suggested that auto mode does at least sometimes correctly determine that the checkpoint fats should be run

rickettmwork commented:
!pb@dev
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=3f466ebf-a883-46d4-adef-057f162c1187) - :x: completed with errors/failures.)
limit.fat.buckets.to.feature: checkpoint

!pb@dev ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=48a701de-653c-4944-b4b0-01e9211aec70) - :x: completed with errors/failures.)

_Note: Target locations of links might be accessible only to IBM employees._

_Note: Target locations of links might be accessible only to IBM employees._

websterc87 reviewed (approved):
Criu is now happily installed on all ubuntu22 children which means this is correctly covered elsewhere. As discussed we will correct any quality gate hole as we spot them over time.
The continuous build is designed to catch anything that slips through.

I have confirmed the PB ran the checkpoint fats in checkpoint mode (180 tests rather than just 2)

rickettmwork commented:
Confirmed.  Checkpoint fats ran correctly in both PBs and all failures are known, unrelated, problems
